### ISSUE TITLE
Cannot load Nvidia sdk onto SCC login node

### ISSUE THREAD

**Issue by eth1030** [md-hpc/cuda#4]
Cannot load Nvidia sdk onto SCC login node 
```
[eth@scc2 cuda]$ module load nvidia-hpc
Lmod has detected the following error:  Unable to load module because of error when evaluating modulefile:
     /share/module.8/programming/nvidia-hpc/2023-23.5.lua: [string "..."]:60: unexpected symbol near '/'
     Please check the modulefile and especially if there is a line number specified in the above message 
While processing the following module(s):
    Module fullname       Module Filename
    ---------------       ---------------
    nvidia-hpc/2023-23.5  /share/module.8/programming/nvidia-hpc/2023-23.5.lua
```

**Comment by eth1030** [comment]
solved by loading `module load cuda/11.3`

**Comment by eth1030** [comment]
solved by loading `module load cuda/11.3`
